src/app/config/setup/discover-octoprint/discover-octoprint.component.html

```diff
@@ -56,14 +56,3 @@
     search again
   </div>
 </div>
-
-<div class="confirm__wrapper" *ngIf="urlSplit">
```

Comment:
Was all this unused?

Reply:
More or less, yes -- I had never seen it pop up on my own setup!